### PR TITLE
UUID campaign creation 

### PR DESCRIPTION
**Pull request by wildmouse**
- Add UUID campaign creation flow
- UUID URLs can be copied to clipboard

### CODE REVIEW

File: frontend/packages/react-app/src/components/organisms/UUIDDistributionTargets/index.tsx [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+      <Box p={4}>
+        <Box mt={4}>
+          <Typography variant={"h4"}>
+            1. Input distribution target quantity
+          </Typography>
+        </Box>
+        <>
+          <Box my={4} style={{ textAlign: "center" }}>
+            <Typography>Distribution target quantity</Typography>
+          </Box>
```
Comment (toshiakitakase) [inline frontend/packages/react-app/src/components/organisms/UUIDDistributionTargets/index.tsx:29]: IMO: I thought that "number of unique URLs" would be easier to understand than "target quantity".

File: frontend/packages/react-app/src/components/organisms/UUIDURLList/index.tsx [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+                {uuidState.rawTargets.length} campaign URLs{" "}
+              </Typography>
+              <Typography display="inline">
+                are generated for your fans.
+              </Typography>
+              <Typography>
+                Copy and distribute it!(Please note, this url will be deleted
+                after you move from this page, so please be careful. )
+              </Typography>
+            </Box>
```
Comment (toshiakitakase) [inline frontend/packages/react-app/src/components/organisms/UUIDURLList/index.tsx:47]:
```
A campaign is created successfully and 100 campaign URLs are generated for your fans.
Copy and distribute it! Please note, this URL will be deleted after you move from this page, so please be careful.
```

**Comment by toshiakitakase** [comment]
I couldn't see `Campaign Name`, `Start Date` and `End Date`
Screen Shot 2021-02-02 at 15 39 37

**Comment by wildmouse** [comment]
> I couldn't see `Campaign Name`, `Start Date` and `End Date`
> Screen Shot 2021-02-02 at 15 39 37

Subgraph is not able to index UUID distributor info currently and so this is another issue. So please skip this.

**Comment by wildmouse** [comment]
Comments fixed.

File: frontend/packages/react-app/src/components/organisms/UUIDURLList/index.tsx [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+        `${window.location.origin}/explore/${tokenAddress}/campaigns/${uuid}`
+    )
+    .join("\n");
+
+  return (
+    <Box mt={2}>
+      <Paper>
+        <Box p={4}>
+          <Box m={"auto"} width={[3 / 4]} mt={2}>
+            <Typography variant={"h4"}>4. UUID URL List</Typography>
```
Comment (toshiakitakase) [inline frontend/packages/react-app/src/components/organisms/UUIDURLList/index.tsx:31]: URL List

File: frontend/packages/react-app/src/components/templates/CreateUUIDCampaignPageTemaplate/index.tsx [hunk 4]
```diff
@@ -0,0 +1,82 @@
...
+    <Box mt={5}>
+      <Container>
+        {!active ? (
+          <Box>
+            <WalletConnect />
+          </Box>
+        ) : (
+          <Box>
+            <Box my={1}>
+              <Typography variant={"h3"}>UUID Campaign</Typography>
```
Comment (toshiakitakase) [inline frontend/packages/react-app/src/components/templates/CreateUUIDCampaignPageTemaplate/index.tsx:46]: URL Campaign

**Review by toshiakitakase — APPROVED** [review]
LGTM except for the above comments.

**Pull request merged**